### PR TITLE
Add recipe for ozzy-pic

### PR DESCRIPTION
**Pull request by mtrocadomoreira**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mtrocadomoreira** [comment]
I'm willing to be listed as a maintainer.

**Comment by mtrocadomoreira** [comment]
@conda-forge/python, ready for review!

**Comment by mtrocadomoreira** [comment]
@conda-forge/help-python, ready for review!

**Comment by mtrocadomoreira** [comment]
@conda-forge-admin, please ping team

File: recipes/ozzy-pic/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+python_min:
```
Comment (ocefpaf) [inline recipes/ozzy-pic/conda_build_config.yaml:1]: This is fine but I prefer the line

```
{% set python_min = python_min|default("3.8") %}
```

in the recipe b/c it is easier to remember when to update it.

File: recipes/ozzy-pic/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    - numpy-groupies >=0.11.2,<0.12.0
+    - numbagg >=0.8.2,<0.9.0
+
+test:
+  imports:
+    - ozzy
+  commands:
+    - pip check
+  requires:
+    - pip
```
Comment (ocefpaf) [inline recipes/ozzy-pic/meta.yaml:49]:
```suggestion
  requires:
    - python {{ python_min }}
    - pip
```

File: recipes/ozzy-pic/conda_build_config.yaml [hunk 3]
```diff
@@ -1,2 +1 @@
-python_min:
-  - 3.10
\ No newline at end of file
+{% set python_min = python_min|default("3.10") %}
```
Comment (ocefpaf) [inline recipes/ozzy-pic/conda_build_config.yaml]: Sorry, I was not clear, what you had here was correct for the conda_build_config.yaml. If you want to get rid of this file you should add that line at the top of the recipe.yaml file.

Comment (mtrocadomoreira) [reply recipes/ozzy-pic/conda_build_config.yaml]: Hi! Thanks for the suggestions, first of all. No worries, I figured it out!

File: recipes/ozzy-pic/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    - numpy-groupies >=0.11.2,<0.12.0
+    - numbagg >=0.8.2,<0.9.0
+
+test:
+  imports:
+    - ozzy
+  commands:
+    - pip check
+  requires:
+    - python {{ python_min }}
```
Comment (mtrocadomoreira) [inline recipes/ozzy-pic/meta.yaml:50]: Thanks for the suggestions @ocefpaf! I did have this line before, but when I include it I get errors in the docker build tests in all three operating systems. It seems like ipython is responsible. Any ideas about how to deal with this?

I'm not sure whether you can access the error logs of the Azure automatic tests (sorry, bit of a noob here), so please let me know if I should copy the error messages to the conversation.

File: recipes/ozzy-pic/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,61 @@
+{% set name = "ozzy-pic" %}
+{% set version = "1.0.5" %}
+{% set python_min = python_min|default("3.10") %}
```
Comment (ocefpaf) [inline recipes/ozzy-pic/meta.yaml:3]: Let's try just this:

```suggestion
{% set python_min = "3.10" %}
```

**Pull request merged**